### PR TITLE
Blog app - views

### PR DESCRIPTION
**Pull request by a-laarabi**
# What?
In this pull request :

- Use preprocessed HTML file with embedded Ruby code.
- Use layouts and templates for shared content.
- Implement the design from the sneak peek wireframes.

## Cheers and happy reviewing 🥇

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        <img src="<%= user.photo %>" alt="<%=user.name %> thumbnail" />
+      </div>
+      <div class="user_info">
+        <h3 class="user_name"><%= link_to user.name, user_path(user) %></h3>
+        <p class="postNumber">Number of Posts: <%= user.posts_counter || 0 %></p>
+      </div>
+    </li>
+    <% end %>
+  </ul>
+</section>
```
Comment (tresorsawasawa) [inline app/views/users/index.html.erb:15]: - [x] Kindly make sure the index view page matches the expected [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_root_users.png) as required for the project, each element should be placed at his correct place.
### Here is your styling:
Screenshot from 2022-11-04 08-45-27

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<section>
+<div class='one_user'>
+    <div class='profile'>
+      <img src='<%= @user.photo %>' alt='<%= @user.name %> thumbnail' />
+    </div>
```
Comment (tresorsawasawa) [inline app/views/users/show.html.erb:5]: - [ ] Kindly make sure you fix the overlapping profile image on each **[wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)** to match the required design. Right now, the profile image on each wireframe makes your design not match the expected result.
You can see on the image of the previous comment.

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        <% end %>
+        <h1 class="user-head"><%= post.title %></h1>
+        <p>post description: <span><%= post.text%></span> </p>
+        <p class="user-para">Number of comments: <%= post.comments_counter || 0 %></p>
+        <p class="user-para">Number of likes: <%= post.likes_counter || 0 %></p>
+          
+      </div>
+    <% end %>
+  <button class="list_btn">pagination</button>
+</section>
```
Comment (tresorsawasawa) [inline app/views/posts/index.html.erb:25]: - [ ] Kindly make sure the `app/views/posts/index.html.erb` page sticks as much as possible to its expected **[wireframe design](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_page.png)**

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @a-laarabi 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arslanbisharat** [comment]
Hi @a-laarabi,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**